build.gradle

```diff
@@ -5,7 +5,6 @@ plugins {
 
     id 'com.diffplug.spotless' version '6.10.0'
     id 'com.github.spotbugs' version '5.0.12'
-    id 'org.gradle.test-retry' version '1.4.1'
```

Comment:
I don't think test-retry requires GE, can we keep that? That was very useful.

Reply:
looks like you're right, adding that back in